README.rst

```diff
@@ -61,6 +61,19 @@ functions, which take funcargs, does not work. Please use
       res = yield threads.deferToThread(os.listdir, tmpdir.strpath)
       assert res == []
 
+
+async_callbacks
+=================
+Using ``twisted.internet.defer.ensureDeferred`` as a decorator for test
+functions, which take funcargs, does not work. Please use
+``pytest_twisted.async_callbacks`` instead::
+
+  @pytest_twisted.async_callbacks
+  async def test_some_stuff(tmpdir):
```

Comment:
Some naming bikeshed: my instinct was to call this "ensureDeferred" because there's "inlineCallbacks" vs "pytest.inlineCallbacks" already. But I guess on the "con" side is that `ensureDeferred` isn't actually a decorator in Twisted like `@inlineCallbacks` is .....

Reply:
In case we needed more sheds, the lowerCamel makes me itchy and I was tempted to add an `@inline_callbacks` option.  :]

I can go with whatever...  I don't have the energy to come up with a reason to care right now.

Reply:
hmm, yeah those are somewhat more descriptive of what's really going on. *And* shorter :+1: :)

Reply:
More back and forth on IRC later and we get to `async_await` and an alias of `async_yield` for `inlineCallbacks`.